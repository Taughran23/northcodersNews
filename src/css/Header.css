.header {
    background-color: black;
    border-width:30px;
    margin-left: 75px;
    margin-right:75px;
    margin-top:50px;
    margin-bottom:50px;
    position:relative
}

@media screen and (max-width:1000px) {
    .header {
      margin-left: 0px;
      margin-right:0px;
      margin-top:0px;
      margin-bottom:10px
    }
}