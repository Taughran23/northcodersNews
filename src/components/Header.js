import React from 'react';
import '../css/Header.css';

const Header = () => (
  <section>
    <div className='header'>
      <img src={'https://northcoders.com/images/logos/learn_to_code_manchester_rw_second.png'} alt='Northcoders'/>
    </div>
  </section>
);

export default Header;