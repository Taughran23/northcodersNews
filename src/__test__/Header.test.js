import React from 'react';
import Enzyme , {shallow} from 'enzyme';
import Header from '../components/Header';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import expect from 'expect';

Enzyme.configure({ adapter: new Adapter() });

describe('<Header />', () => {
    
  it('should render correctly', () => {
    const component = shallow(<Header />);
    expect(shallowToJson(component)).toMatchSnapshot();
  });
  it('it renders 1 <Header /> component', () => {
    const component = shallow(<Header />);
    expect(component).toHaveLength(1);
  });
});