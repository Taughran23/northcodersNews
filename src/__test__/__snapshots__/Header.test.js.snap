// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render correctly 1`] = `
<section>
  <div
    className="header"
  >
    <img
      src="https://northcoders.com/images/logos/learn_to_code_manchester_rw_second.png"
    />
  </div>
</section>
`;
